staking-cli/src/deploy.rs

```diff
@@ -76,7 +74,7 @@ impl TestSystem {
         let token = EspToken::new(*token_proxy.address(), provider.clone());
 
         // `StakeTable.sol`
-        let stake_table_impl = StakeTable::deploy(provider.clone()).await?;
+        let stake_table_impl = StakeTableV2::deploy(provider.clone()).await?;
```

Comment:
when this gets pushed to main, would the public-facing staking cli have these changes? if so, will we need to upgrade StakeTableV2 before merging this into main?

Reply:
At this line here it's only affecting the staking-cli tests. My goal is to write staking-cli and GCL code so it can all work with both versions of the contract.